CHANGELOG.md

```
@@ -40,37 +39,26 @@
   * Dependency update: commons-lang3 -> 3.14.0
   * Dependency update: gson -> 2.11.0
 
-#### 10.1.6 - May 2024
+#### 9.4.5 - May 2024
   * (PW-1875) Changed the BusinessApplicationHeaderV01 marshaller to always use Zulu timezone with "Z" indicator
-
-#### 10.1.5 - April 2024
-  * Update jaxb dependency from com.sun.xml.bind:jaxb-impl:4.0.2 to 4.0.5 containing several jaxb bugfixes
-
-#### 10.1.4 - January 2024
+ 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)